pavelib/servers.py

```diff
@@ -2,24 +2,28 @@
 Run and manage servers for local development.
 """
 from __future__ import print_function
-import sys
 import argparse
 from paver.easy import *
+
+from .assets import collect_assets
 from .utils.cmd import django_cmd
 from .utils.process import run_process, run_multi_processes
 
 
 DEFAULT_PORT = {"lms": 8000, "studio": 8001}
 DEFAULT_SETTINGS = 'devstack'
+OPTIMIZED_SETTINGS = "devstack_optimized"
+OPTIMIZED_ASSETS_SETTINGS = "test_static_optimized"
 
 
-def run_server(system, settings=None, port=None, skip_assets=False, contracts=False):
+def run_server(system, settings=None, asset_settings=None, collect_static=False, port=None, no_contracts=False):
     """
     Start the server for the specified `system` (lms or studio).
     `settings` is the Django settings module to use; if not provided, use the default.
+    `collect_static` defaults to False, but if True then static files will be collected.
+    `asset_settings` is the settings to use when generating assets; if not provided, assets are not generated.
     `port` is the port to run the server on; if not provided, use the default port for the system.
-
-    If `skip_assets` is True, skip the asset compilation step.
+    `no_contracts` is true if contracts are not to be enabled. The default is to include contracts.
```

Comment:
I believe that this is related to enabling PyContracts in the code:

https://andreacensi.github.io/contracts/

This already existed, but I just standardized the behavior (my tests discovered that they were inconsistent about what the default was and whether the option was even supported).